Plugin fails with "error": "Sorry, something bad happened!"

If I try to access the Pluginpage in the Adminboard the Server logs the following Message:

```
[2014-12-11 17:11:59.817] [ERROR] console - TypeError: Object #<Namespace> has no method 'clients'
at exports.registerRoute (/var/lib/etherpad-lite/etherpad-lite/node_modules/ep_message_all/index.js:41:35)
at async.series.results (/var/lib/etherpad-lite/etherpad-lite/src/node_modules/async/lib/async.js:610:21)
at _asyncMap (/var/lib/etherpad-lite/etherpad-lite/src/node_modules/async/lib/async.js:249:17)
at async.eachSeries.iterate (/var/lib/etherpad-lite/etherpad-lite/src/node_modules/async/lib/async.js:149:13)
at async.eachSeries.iterate (/var/lib/etherpad-lite/etherpad-lite/src/node_modules/async/lib/async.js:160:25)
at _asyncMap (/var/lib/etherpad-lite/etherpad-lite/src/node_modules/async/lib/async.js:251:21)
at async.series.results (/var/lib/etherpad-lite/etherpad-lite/src/node_modules/async/lib/async.js:615:34)
at exports.registerRoute.args.app.get.async.series.render_args.message (/var/lib/etherpad-lite/etherpad-lite/node_modules/ep_message_all/index.js:36:9)
at async.series.results (/var/lib/etherpad-lite/etherpad-lite/src/node_modules/async/lib/async.js:610:21)
at _asyncMap (/var/lib/etherpad-lite/etherpad-lite/src/node_modules/async/lib/async.js:249:17)
```

and the site is just:

```
{
   "error": "Sorry, something bad happened!"
}
```

I use actual develop branch from etherpad-lite

--- a/index.js
+++ b/index.js
@@ -16,14 +16,12 @@ exports.eejsBlock_adminMenu = function(hook_name, args, cb) {
 exports.registerRoute = function (hook_name, args, cb) {
   args.app.get('/admin/message_to_all', function(req, res) {
     var message = req.query.message;
-	
     async.series([
       function(callback){
         if (message && message != "") {
-        var clients = socketio.sockets.clients();
-
-          for(var i = 0; i < clients.length; i++) {
-            clients[i].json.send({type: "COLLABROOM",
+          var clients = socketio.sockets.adapter.nsp.connected;
+          for(var client in clients) {
+            clients[client].send({type: "COLLABROOM",
               data:{
                 type: "shoutMessage",
                 payload:{
@@ -38,7 +36,7 @@ exports.registerRoute = function (hook_name, args, cb) {
       function(callback){
         var render_args = {
           message: message,
-          users: socketio.sockets.clients().length
+          users: socketio.sockets.adapter.nsp.connected.length
         };
         res.send( eejs.require("ep_message_all/templates/admin/shout.html", render_args) );
         callback();

